src/components/ImportRecordsFlow/stepTwo/__test__/previewTable.spec.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { mount, shallow } from 'enzyme';
+import PreviewTable from '../previewTable';
+
+const data = [
+    { First_Name: 'John', Last_Name: 'Doe' },
+    { First_Name: 'Marie', Last_Name: 'Sun' },
+    { First_Name: 'Jane', Last_Name: 'Doe' },
+];
+const columns = ['First_Name', 'Last_Name'];
+
+describe('<Preview />', () => {
```

Comment:
Oh, I see, great. Only need to add the test for when isLoading is false and put the PreviewTable in another file, also change the describe name to `<PreviewTable />`